backend/sources/facebook/connector/src/main/java/co/airy/core/sources/facebook/ChannelsController.java

```diff
@@ -104,4 +106,10 @@ public ChannelsController(Api api, Stores stores) {
             return ResponseEntity.status(HttpStatus.SERVICE_UNAVAILABLE).build();
         }
     }
+
+    @Override
+    public Health health() {
```

Comment:
This indicator should already be on the `Stores` class

Reply:
will move all of them there! It's a better place :)